test/helm_task_test.go

```diff
@@ -157,7 +157,7 @@ func getGoHelloworldGitResource(namespace string) *v1alpha1.PipelineResource {
 			Params: []v1alpha1.Param{
 				v1alpha1.Param{
 					Name:  "Url",
-					Value: "https://github.com/pivotal-nader-ziada/gohelloworld",
+					Value: "https://github.com/knative/build-pipeline",
```

Comment:
for the record, this is going to be pretty weird, and maybe confusing b/c it'll be pulling from master XD aw well

Reply:
I open to changing it, but not sure to what? :) we got comments about the external repo. 

Is it an option to get a helloworld repo in the knative org for testing?

Reply:
it's fine to leave it for now, seems like there are cons to all the options